code/modules/shuttles/docking_beacon.dm

```diff
@@ -312,11 +312,11 @@
 			break
 		area_turf = get_step(area_turf, dir)
 
-	// Otherwise, use the planetary or world area.
+	// Otherwise, use the level or world area.
 	if(!base_area)
-		var/obj/effect/overmap/visitable/sector/exoplanet/planet = global.overmap_sectors[num2text(z)]
-		if(istype(planet))
-			base_area = ispath(planet.planetary_area) ? planet.planetary_area : planet.planetary_area.type
+		var/datum/level_data/LD = SSmapping.levels_by_z[z]
```

Comment:
Is that because of that one guy that came in the chat and had a fit about vars being initials?

Personally, I feel initials, when used for a thing you'd just name by it's class name anyways ensure we don't get name collisions with actually important wider scope vars. 
Local, atom, and global vars don't have any naming conventions and can easily clash, so I'd be kinda wary of doing that, unless we get some naming conventions in place as well.